util/msys2_install.sh

```diff
@@ -63,11 +67,37 @@ else
     done
 fi
 
+pacman -q -Q mingw-w64-x86_64-avrdude
```

Comment:
Avrdude should be installed on the newer versions: 
https://github.com/qmk/qmk_firmware/blob/master/util/msys2_install.sh#L19